packages/cover-traffic-daemon/src/index.ts

```diff
@@ -56,6 +56,11 @@ const argv = yargs(process.argv.slice(2))
     string: true,
     demandOption: true
   })
+  .option('provider', {
+    string: true,
+    describe: 'A custom RPC provider to be used for your HOPR node to connect to blockchain',
+    default: 'http://127.0.0.1:8535/'
```

Comment:
Default should be the one defined in our environment.

Reply:
Removed the default, because the default value should be populated per environment.